package.json

```diff
@@ -18,10 +18,12 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
+    "@storybook/react": "^7.0.2",
```

Comment:
This is actually not good. You should leave the packages as they were because:

1. Everything is working.
2. You are opening us up to unknown trouble in the future.

As I mentioned to @ISHIMWE-Aime in this #28 , don't switch around the order of packages. Especially because we are using a beta version, and the behavior will start becoming bizarre.

If you are facing an es-lint error in imports, it's because you are importing a package when you should be importing types. Check the review I left for Aime. It will elaborate.